lib/vectortilefeature.js

```diff
@@ -192,6 +197,57 @@ VectorTileFeature.prototype.toGeoJSON = function(x, y, z) {
     return result;
 };
 
+VectorTileFeature.prototype.toOffsetsJSON = function() {
```

Comment:
How about **toTileCoordsJSON**?

Reply:
Yes, that is much better. I didn't take into account that they are not offsets anymore, but tile coordinates.